test/integration/routes/payment.test.js

```diff
@@ -0,0 +1,94 @@
+const db = require('../../../server/models')
+const createServer = require('../../../server/index')
+
+describe('API', () => {
+  let server
+
+  beforeAll(async () => {
+    await db.schedule.bulkCreate([
+      { scheduleId: 1, claimId: 'MINE123', paymentDate: '2020-03-01 14:30' },
+      { scheduleId: 2, claimId: 'MINE123', paymentDate: '2020-04-01 14:30' },
+      { scheduleId: 3, claimId: 'MINE124', paymentDate: '2020-05-01 14:30' }
+    ])
+    await db.payment.bulkCreate([
+      { claimId: 'MINE123', value: 150.50 },
+      { claimId: 'MINE124', value: 50.75 }
+    ])
+  })
```

Comment:
I think it might make the test cleaner if some of the sample data is held in variables, i.e.
`const claim123 = { claimId: 'MINE123', value: 150.50 }`
So the expectations are against an existing constant, rather than a newly defined constant.

It's a matter of personal preference really, and there's an added complication of the date format being slightly different for the `bulkCreate`

Reply:
I'm going to leave this as is for now - I prefer my inputs and expected outputs to be separate as it feels more like double entry book-keeping.

Maybe we can use it as a COP discussion point.